packages/model/src/team.ts

```diff
@@ -42,6 +45,8 @@ export interface TeamMember {
 
 export type TeamDataObject = Omit<TeamCreateRequest, 'applicationNumber'> & {
   id: string;
+  active: boolean;
+  inactiveSince?: string | null;
```

Comment:
I wonder if we need to surface two fields at all, like I mentioned in the other PR I think this could be a single field in the CMS, definitely in the data object and probably even in the response type, although if we need to add a boolean for simplicity we can.